mlrun/common/schemas/alert.py

```diff
@@ -120,3 +120,25 @@ class AlertConfig(pydantic.BaseModel):
 class AlertsModes(StrEnum):
     enabled = "enabled"
     disabled = "disabled"
+
+
+class AlertTemplate(
+    pydantic.BaseModel
+):  # Template fields that are not shared with created configs
+    template_id: int = None
+    template_name: str
+    template_description: Optional[str] = (
+        "String explaining the purpose of this template"
+    )
+
+    # A property identifying templates that were created by the system and cannot be modified/deleted by the user
```

Comment:
# A property that identifies templates that were created by the system and cannot be modified/deleted by the user